### PR TITLE
Updated Categories as per 0.9.0

### PR DESCRIPTION
**Pull request by joseph619**
All sections also need updated videos (which have been removed in this edit). Bottom two have been kept which also need to be replaced.

### CODE REVIEW

**Comment by Soitora** [comment]
I believe this error is due to you referencing a navigation item that does not exist

File: src/help/guides/categories.md [hunk 1]
```diff
@@ -9,57 +9,48 @@ lang: en-US
...
-<figure class="centered">
-	<video muted loop controls :poster="$withBase('/assets/media/video-guide-category-edit.png')" height="500" controlslist="nodownload noremoteplayback" preload="none" loading="lazy" crossorigin="use-credentials">
-		<source :src="$withBase('/assets/media/video-guide-category-edit.webm')" type="video/webm"/>
-	</video>
-</figure>
-:::
+1. Go to <Navigation item="more"/> → <Navigation item="settings"/> → <Navigation item="library"/>.
+1. Tap on the **Edit categories**.
+1. Long press a category.
+1. Click <Navigation item="edit"/> on top right.
```
Comment (Soitora) [inline src/help/guides/categories.md:26]:
```suggestion
1. Click **EDIT** on top right.
```
Probalby it's this that does not exist.

Comment (joseph619) [reply src/help/guides/categories.md:26]: Hmm I am not sure what the name is for the pencil icon, upon Long pressing it was labelled as Edit on the app.

Comment (Soitora) [reply src/help/guides/categories.md:26]: What I have added for the <Navigation> element you can find here: https://github.com/tachiyomiorg/website/blob/master/src/.vuepress/components/Navigation.vue

Comment (joseph619) [reply src/help/guides/categories.md:26]: I added it under `/* Actions */`, should I change it to under `/* Misc */`?

Comment (Soitora) [reply src/help/guides/categories.md:26]: No Actions is probably better

Comment (joseph619) [reply src/help/guides/categories.md:26]: Hmm, so what do I do now?

Comment (Soitora) [reply src/help/guides/categories.md:26]: > Hmm, so what do I do now?

Rebase your branch on top of the current upstream/master, probably

**Pull request merged**